src/shared/styles/theme.tsx

```diff
@@ -446,7 +446,8 @@ export const theme = createTheme({
           borderRadius: variables.borderRadius.md,
           fontSize: variables.font.size[size === 'small' ? 'md' : 'lg'],
           fontWeight: variables.font.weight.regular,
-          letterSpacing: size === 'small' ? 0 : variables.font.letterSpacing.xxl,
+          letterSpacing:
+            size === 'small' ? variables.font.letterSpacing.lg : variables.font.letterSpacing.xxl,
```

Comment:
Just making sure. Is `lg` what we want for this one? `sm` is also available, representing `0.01rem`. Currently, this would give us `0.025rem`.